rpc/service.go

```diff
@@ -113,6 +113,7 @@ func GetServer(env cedar.Environment, conf CertConfig) (*grpc.Server, error) {
 		opts,
 		grpc.UnaryInterceptor(aviation.ChainUnaryServer(unaryInterceptors...)),
 		grpc.StreamInterceptor(aviation.ChainStreamServer(streamInterceptors...)),
+		grpc.MaxRecvMsgSize(5e9),
```

Comment:
should this be configurable? this is maxint right? shouldn't we set this at like some small number of gigs to avoid ooming?